src/core/seat/cursor.cpp

```diff
@@ -208,12 +211,15 @@ wf_cursor::wf_cursor()
     wl_signal_add(&cursor->events.motion_absolute, &motion_absolute);
     wl_signal_add(&cursor->events.axis, &axis);
 
-    init_xcursor();
-
     config_reloaded = [=] (signal_data*) {
         init_xcursor();
+        auto section = core->config->get_section("input");
```

Comment:
You need to get options just once, `wf_option` is actually an alias for a shared_ptr.